frontend/src/hooks/analytics/analytics.ts

```diff
@@ -0,0 +1,44 @@
+import Analytics from "analytics-node";
+import { uniqueId } from "lodash";
+import { getConfig } from "../../libraries/config";
+
+export interface Dict {
+  [key: string]: any;
+}
+
+let segment: Analytics;
+let anonymousId: string;
+
+function init() {
+  if (segment)
```

Comment:
Do you mind adding `{ }` around to keep code unified through the projects (I didn't have time to add linting yet)